StreamDeckPlugin/Program.cs

```diff
@@ -12,8 +12,12 @@ class Program {
 #endif
 
             var requestHandler = new TcpRequestHandler();
-            new ReceiveSocketService(requestHandler).StartListening(StreamDeckTcpInfo.Port);
-            new RegisterForUpdatesService(requestHandler).RegisterForUpdates();
+
+            var receiveSocketService = new ReceiveSocketService(requestHandler);
```

Comment:
Minor: I would consider adding comments here to avoid future cleanups causing a regression.

Reply:
Done